app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ExpensesController < ApplicationController
+  
+  def create
+    begin
+      expense = Expense.create(expense_params)
+      render status: :created, json: expense.to_json
+    rescue
+      render status: 400
+    end
+  end
+
+  def destroy
+    begin
+      params.require(:id)
+      Expense.destroy_by(id: params[:id])
```

Comment:
Similar to the other comment I left in `update`, let's first find the record using `Expense.find_by!` to ensure that it exists. Then, catch for `ActiveRecord::RecordNotFound` when it doesn't exist and return `404` (this is the rescue statement).